src/igvfd/types/analysis_step.py

```diff
@@ -47,6 +46,49 @@ def unique_keys(self, properties):
         keys.setdefault('analysis_step:name', []).append(self._name(properties))
         return keys
 
+    @calculated_property(schema={
```

Comment:
I think in this section, you will require `define=True`, so that a calculated property can be used by another calculated property (like how "workflows" below is relying on this calculated "analysis_step_versions")